### PR TITLE
Border on details page

### PR DESCRIPTION
**Pull request by zara19013**
borders country details fetch and display

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -50,11 +51,14 @@ function App() {
                 </>
               }
             />
+              <Route path="/details" element={<DetailsPage />} />
+            <Route path="/border-details" element={<DetailsPage />} />
+            </Routes>
```
Comment (asad-shayansolution) [inline src/App.js]: Use prettier extension and format your code before commit

File: src/components/homepage/filter.css [hunk 2]
```diff
@@ -1,142 +1,130 @@
+.FilterBox {
```
Comment (asad-shayansolution) [inline src/components/homepage/filter.css:1]: why these classname are capital?

**Review by asad-shayansolution — APPROVED** [review]
ok

File: src/components/Detail/Detail.js [hunk 3]
```diff
@@ -11,15 +10,36 @@ function DetailsPage() {
...
       .get(`https://restcountries.com/v3.1/alpha/${countryCode}`)
-
       .then((response) => {
         const fetchedCountryData = response.data[0];
         setCountryData(fetchedCountryData);
+
+        const borderPromises = fetchedCountryData.borders?.map(
+          (borderCountryCode) =>
+            axios.get(
+              `https://restcountries.com/v3.1/alpha/${borderCountryCode}`
```
Comment (imran-munawar) [inline src/components/Detail/Detail.js]: add base URL in env

**Review by imran-munawar — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (zara19013) [reply src/App.js]: ok

File: src/components/homepage/filter.css (hunk 2 above)
Comment (zara19013) [reply src/components/homepage/filter.css:1]: i don't get your point!

**Review by asad-shayansolution — APPROVED** [review]

**Review by imran-munawar — APPROVED** [review]

**Pull request merged**